src/pages/loginRegister.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react"
+import "./loginRegister.css"
+import { LoginForm } from "../components/loginForm"
+import { RegisterForm } from "../components/registerForm"
+
+function LoginRegister(props) {
+  const loginForm = <LoginForm setLoggedIn={props.setLoggedIn} />
+  const [form, setForm] = useState(loginForm)
```

Comment:
Saving JSX into the state is not recommended instead, save values in the state and take decisions based on values.

```
const [activeFormType, setActiveFormType] = useState('login')
```